bpm/bonita-api/bonita-common-api/src/main/java/org/bonitasoft/engine/bpm/contract/FileInputValue.java

```diff
@@ -32,9 +32,15 @@
 
     private String fileName;
     private byte[] content;
+    private String contentType;
 
     public FileInputValue(String fileName, byte[] content) {
+        this(fileName, null, content);
```

Comment:
This can be used from groovy script so it is to keep legacy happy and avoid migration.